test/replacescopename.js

```diff
@@ -1,20 +1,20 @@
 'use strict'
 
-const replaceScopeName = require('../src/replaceScopeName')
+const replaceScopeName = require('../src/replacescopename')
 const test = require('ava')
 
 test('Return empty string if none available', t => {
   t.is(replaceScopeName(), '')
 })
 
 test('Return same name if not scoped', t => {
-  t.is(replaceScopeName('poopie'), 'poopie')
+  t.is(replaceScopeName('myapp'), 'myapp')
```

Comment:
@malept We should probably change the names of all the examples. Poopie is the name used everywhere in all installers modules